geppetto.js/geppetto-client/js/geppettoProject/ProjectsController.js

```diff
@@ -115,9 +115,13 @@ define(function (require) {
         }
       };
 
-    GEPPETTO.on(GEPPETTO.Events.Experiment_loaded, function () {
-      // fetch at least every time and experiment is loaded
-      GEPPETTO.ProjectsController.refreshUserProjects();
-    });
+    // FIXME: maybe not a fixme, just to keep track that this has been moved to the middleware
```

Comment:
🤔  does moving it to middleware break anything?
If it doesn't require an action, I would rather remove the comment, someone that looks for this action can find it in the middleware by running a search

Reply:
another experiment related thing cominq from @ddelpiano original refactoring